autoload/ledger.vim

```diff
@@ -393,6 +393,18 @@ function! ledger#align_amount_at_cursor()
   endif
 endf!
 
+function! ledger#autocomplete_and_align()
+  if pumvisible()
+    return "\<c-n>"
+    " See http://stackoverflow.com/questions/23323747/vim-vimscript-get-exact-character-under-the-cursor
```

Comment:
@kljohann I contacted him and he licensed it under GPL for us, so we're completely in the clear.  Anything blocking merger now?